release-schema.json

```
@@ -5,6 +5,15 @@
       "title": "Sustainability",
       "description": "Information about how the contracting process or lot incorporates Sustainable Public Procurement.",
       "properties": {
+        "description": {
+          "title": "Description",
+          "description": "A description of how this contracting process or lot incorporates Sustainable Public Procurement. Structured information should be provided in `.goal` and `.strategies`.",
```

We typically do not provide instructions on how to use other fields within a given field.

Use same article as definition's description.

```suggestion
          "description": "A description of how the contracting process or lot incorporates Sustainable Public Procurement.",
```

---

The 'Structured information...' part is taken from the [style guide template for description fields](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#descriptions). Should we update the style guide?

---

Yes, we should update the guide. I think only the requirements extension uses this phrasing (we can create an issue to also remove that).